satpy/demo/fci.py

```diff
@@ -17,12 +17,14 @@
 # satpy.  If not, see <http://www.gnu.org/licenses/>.
 """Demo FCI data download."""
 
-import tarfile
 import pathlib
+import tarfile
 import tempfile
-from . import utils
+
 from satpy import config
 
+from . import utils
+
```

Comment:
Are the rules for import orders written down somewhere?  The changes on the import order seems to go beyond what is formulated in PEP 8.

Reply:
It is not a PEP8 thing and is configured in pyproject.toml for the "isort" tool.